crux-http-client/src/crux/remote_api_client.clj

```diff
@@ -188,23 +189,19 @@
   (hasTxCommitted [_ submitted-tx]
     (api-request-sync (str url "/tx-committed") submitted-tx))
 
-  (newTxLogContext [_]
-    (->RemoteApiStream (atom [])))
-
-  (txLog [_ tx-log-context from-tx-id with-documents?]
+  (openTxLogIterator [this from-tx-id with-ops?]
     (let [params (->> [(when from-tx-id
                          (str "from-tx-id=" from-tx-id))
-                       (when with-documents?
-                         (str "with-documents=" with-documents?))]
+                       (when with-ops?
+                         (str "with-ops=" with-ops?))]
                       (remove nil?)
                       (str/join "&"))
           in (api-request-sync (cond-> (str url "/tx-log")
                                  (seq params) (str "?" params))
                                nil
                                {:method :get
                                 :as :stream})]
-      (register-stream-with-remote-stream! tx-log-context in)
```

Comment:
What happened to this client side resource management? Can't recall if it was actually doing anything useful TBH?

Reply:
I'm not entirely sure what it was for, removed it as you didn't have a tx-log-context anymore so wasn't sure if there was anything else you could've passed into it?

Reply:
pass!

Reply:
My point was more that these streams used to be tracked, potentially to be closed (client side), but can't really recall what that really did practically. There's the lazy query variant as well, if you look at how that works and why and if it uses this.

Reply:
yep - this needs to be closed in the close-fn, pushing a change now